source/stylesheets/elements/_layout.scss

```diff
@@ -0,0 +1,51 @@
+ul {
+  list-style: none;
+  padding: 0rem;
+}
+
+a {
+  text-decoration: none;
+  color: $teal;
+  &:hover {
+    color: $teal-dark;
+  }
+}
+
+img {
+  max-width: 100%;
+}
+
```

Comment:
Your `ul`, `a`, and `img` tags aren't really layout per-say. I'd move these into another partial. I typically call something like this something like "base", as it's styling you get out of the box before applying any other logic to it.